### PR TITLE
Fixed /jobs archive incorrectly displaying

### PR DESCRIPTION
**Pull request by KingCreator11**
Implemented the fix to the following issue:
https://github.com/Zrips/Jobs/issues/1028

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "interactive"
+}
```
Comment (montlikadani) [inline .vscode/settings.json:3]: Please add this file into **.gitignore** and remove from there.

File: pom.xml [hunk 2]
```diff
@@ -11,9 +11,18 @@
...
 	</properties>
 
 	<dependencies>
+		<!-- Boss -->
+		<dependency>
+			<groupId>org.mineacademy.plugin</groupId>
+			<artifactId>Boss-API</artifactId>
+			<version>3.8.0</version>
+			<scope>provided</scope>
+		</dependency>
```
Comment (montlikadani) [inline pom.xml:25]: Remove this dependency including repository.

**Comment by montlikadani** [comment]
https://github.com/Zrips/Jobs/commit/15ae2988e0861c0819d0b732ee668ad17ed28ac1

File: .vscode/settings.json (hunk 1 above)
Comment (KingCreator11) [reply .vscode/settings.json:3]: The changes have been reverted, in the future I'll use a separate branch for my own purposes.

**Pull request closed without merging**